better-politics-mod/events/bpm_union_leadership.txt

```diff
@@ -0,0 +1,61 @@
+namespace = bpm_union_leadership
+
+# State Union Passed
+bpm_union_leadership.1 = { 
+	type = country_event
+	placement = root
+
+	event_image = { video = "gfx/event_pictures/middleeast_police_breaking_door.bk2" }
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/europenorthamerica/political_extremism"
+	icon = "gfx/interface/icons/event_icons/event_newspaper.dds"
+	
+	title = bpm_union_leadership.1.t
+	desc = bpm_union_leadership.1.d
+	flavor = bpm_union_leadership.1.f
+	duration = 3
+
+	trigger = {
+	}
+	
+	immediate = {
```

Comment:
Not 100% sure we should kill the character. Maybe make them an agitator instead?